example/app/internal/model/schema_news_test.go

```diff
@@ -756,3 +756,117 @@ func TestNewsRepositoryBase_UpsertQuery(t *testing.T) {
 		})
 	}
 }
+
+func TestNewsRepositoryBaseTx_Insert(t *testing.T) {
+	s := setup(t)
+	defer s.teardown(t)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 50*time.Second)
+	defer cancel()
+	tx, err := s.news.BeginTx(ctx)
```

Comment:
@ders Separate API was replaced by additional `struct` that has similar API but each statement is executed within the scope of a transaction.